2_observations.yml

```diff
@@ -18,29 +18,38 @@ targets:
       - out_data/reservoir_releases.csv
```

Comment:
These reservoir releases file names need to be updated here as well

Reply:
good catch. Fixed.